geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloDataStoreFactory.scala

```diff
@@ -117,10 +117,9 @@ class AccumuloDataStoreFactory extends DataStoreFactorySpi {
     val modifiedParams = params ++ Map(authsParam.key -> auths.mkString(","))
     authorizationsProvider.configure(modifiedParams)
 
-    val featureEncoding =
-      featureEncParam.lookupOpt[String](params)
-        .map(FeatureEncoding.withName)
-        .getOrElse(FeatureEncoding.AVRO)
+    val featureEncoding = featureEncParam.lookupOpt[String](params)
+      .map(FeatureEncoding.withName)
+      .getOrElse(FeatureEncoding.KRYO)
```

Comment:
yeah, i was thinking that but wasn't sure how often it was going to change... I'll go ahead and add one.